### PR TITLE
Add back link to links component.

### LINKED ISSUE #354: Add back link component.
**Description**
This component does not exist on our component pages.
Documentation is available.
Design is not available yet.

Relates to: https://github.com/wmcadigital/wmn-sprints/issues/21

### PR DESCRIPTION
**Pull request by catiarodriguescosta**
It's inside links.

Fixes #354

### CODE REVIEW

File: src/www/pages/components/links/index.njk [hunk 1]
```diff
@@ -75,6 +75,41 @@
...
+</ul>
+{# How it works #}
+<h3>How it works?</h3>
+<ul>
+  <li>Always place back links at the top of the page.</li>
+  <li>Make sure the link takes users to the previous page they were on, in the state they last saw it.</li>
+  <li>If you cannot do this you should hide the back link. For example, when JavaScript is not available.</li>
+</ul>
+<br><br>
+{% from "wmnds/components/link/link/_link.njk" import wmndsLink %}
```
Comment (gldgrnt) [inline src/www/pages/components/links/index.njk:105]: Remove the two `<br>` tags to be consistent with the rest of the page

**Review by gldgrnt — CHANGES_REQUESTED** [review]
All good except for spacing before component example

**Review by gldgrnt — APPROVED** [review]
Looks good to me now

**Review by daylesalmon — APPROVED** [review]

**Pull request merged**